src/p_saveg.c

```diff
@@ -2693,6 +2695,7 @@ void P_UnArchiveSpecials (void)
           saveg_read_scroll_t(scroll);
           scroll->thinker.function.p1 = (actionf_p1)T_Scroll;
           P_AddThinker(&scroll->thinker);
+          P_AddSideScroller(scroll);
```

Comment:
Only if `scroll->type == sc_side", c.f. `Add_Scroller()`.

Reply:
Fixed in https://github.com/fabiangreffrath/woof/pull/733/commits/c0980d55cd7a0312bda5898b3084497a633f0385